webapp/logging.json

```diff
@@ -6,10 +6,16 @@
             "class": "pythonjsonlogger.jsonlogger.JsonFormatter"
         }
     },
+    "filters": {
+        "default": {
+            "()": "app.logging.AddRequestIdFilter"
+        }
+    },
```

Comment:
Here we define that filter(adding the requestId) and below we are then using it, correct?

Reply:
Correct!